templates/controllers/grid/navigationMenus/remoteUrlNMIType.tpl

```diff
@@ -7,7 +7,7 @@
  *
  * Custom remote Url NMI Type edit form part
  *}
-{fbvFormSection id="NMI_TYPE_REMOTE_URL" class="NMI_TYPE_CUSTOM_EDIT" title="manager.navigationMenus.form.url" for="url" list=true required="true"}
-	{fbvElement type="text" id="url" value=$url maxlength="255" required="true"}
+{fbvFormSection id="NMI_TYPE_REMOTE_URL" class="NMI_TYPE_CUSTOM_EDIT" title="manager.navigationMenus.form.url" for="remoteUrl" list=true required="true"}
+    {fbvElement type="text" multilingual=true id="remoteUrl" value=$remoteUrl maxlength="255" required="true"}
```

Comment:
Watch out for inadvertent changes to indentation; we use tabs throughout our code.